ADR/0031-sprayproxy.md

```diff
@@ -0,0 +1,29 @@
+# 0031 Sprayproxy
+
+Created: 2023-10-24
+
+## Status
+
+Proposed
```

Comment:
Since we're documenting the current implementation, I would put it as 'Accepted'.

Reply:
Agree. That was my feeling as well. Updated.